net-p2p/resilio-sync-bin/resilio-sync-bin-2.5.6.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils systemd tmpfiles unpacker user
+
+MY_PN="${PN/-bin/}"
+BASE_URI="http://linux-packages.resilio.com/${MY_PN}/deb/pool/non-free/r/${MY_PN}/${MY_PN}_${PV}-1_-arch-.deb"
+NAME="rslsync"
+
+DESCRIPTION="Resilient, fast and scalable file synchronization tool"
+HOMEPAGE="https://getsync.com/"
+SRC_URI="amd64? ( ${BASE_URI/-arch-/amd64} )
+	arm? ( ${BASE_URI/-arch-/armhf} )
+	x86? ( ${BASE_URI/-arch-/i386} )"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="pax_kernel"
+RESTRICT="bindist mirror"
+
+S="${WORKDIR}"
+
+QA_PREBUILT="usr/bin/${NAME}"
+
+pkg_setup() {
+	enewgroup ${NAME}
+	enewuser ${NAME} -1 -1 /var/lib/${MY_PN} ${NAME}
+}
+
+src_install() {
+	dobin usr/bin/${NAME}
+	use pax_kernel && pax-mark m "${ED%/}"/usr/bin/${NAME}
+
+	doman usr/share/man/man1/${MY_PN}.1.gz
```

Comment:
This should bring out a big QA warning that `doman` is not allowed on compressed files. You need to decompress it first, e.g. using `unpack`.